.wordlist.txt

```diff
@@ -509,6 +521,7 @@ otherAmountMath
 otherIssuer
 otherMint
 otherPayment
+ourProposal
```

Comment:
Oh, that's too bad that the tags have to be spell-checked as well. Is there any way to exclude some contexts from spelling checks? No? I just thought I'd ask.

Reply:
I think there might be. There's an `ignores` configuration setting that is set to `code` and `pre` currently. The spellchecker is currently running on the input markdown, but if I change it to run on the output html, it might ignore those sections successfully. I'll make an issue.

Reply:
Issue here: https://github.com/Agoric/documentation/issues/321